genai-perf/README.md

```diff
@@ -73,45 +73,34 @@ INSTALLATION
 
 ## Installation
 
-The easiest way to install GenAI-Perf is through
-[Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).
-Install the latest release using the following command:
+The easiest way to install GenAI-Perf is through pip.
+### Install Perf Analyzer (Ubuntu 24.04, Python 3.10+)
 
 ```bash
-export RELEASE="24.10"
-
-docker run -it --net=host --gpus=all  nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
-
-# Check out genai_perf command inside the container:
-genai-perf --help
+pip install genai-perf
 ```
+**NOTE**: you must already have CUDA 12 installed
 
-<details>
 
-<summary>Alternatively, to install from source:</summary>
+<details>
 
-Since GenAI-Perf depends on Perf Analyzer,
-you'll need to install the Perf Analyzer binary:
+<summary>Alternatively, to install the container:</summary>
 
-### Install Perf Analyzer (Ubuntu, Python 3.10+)
+[Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver)
 
-**NOTE**: you must already have CUDA 12 installed
-(checkout the [CUDA installation guide](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html)).
+Install the latest release using the following command:
 
 ```bash
-pip install tritonclient
-
-sudo apt update && sudo apt install -y --no-install-recommends libb64-0d
-```
-
-You can also build Perf Analyzer [from source](../docs/install.md#build-from-source) as well.
+export RELEASE="24.10"
 
-### Install GenAI-Perf from source
+docker run -it --net=host --gpus=all  nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
 
-```bash
-pip install git+https://github.com/triton-inference-server/perf_analyzer.git#subdirectory=genai-perf
+# Check out genai_perf command inside the container:
+genai-perf --help
 ```
 
+You can also build Perf Analyzer [from source](../docs/install.md#build-from-source) to use alongside GenAI-Perf as well.
```

Comment:
This may be beyond the scope of this PR, but we should make sure we are clear when referring to PA. We could have a separate section describing it and describing why you might want to rebuild it. The casual user shouldn't need to know what PA is and how it works within GAP, so reading statements about it without context in the documentation will be more confusing than helpful IMO.

Reply:
That is a good point but out of scope for this since we are trying to get this to a customer today.
In general, we need a pass on all our docs to make them cleaner.